bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/TpeEditorTools.java

```diff
@@ -66,7 +63,6 @@ static ButtonState get(JToggleButton btn) {
             setToolTipText("Switch to browse mode");
             addActionListener(e -> {
                     _current = this;
-                    modeChange(TpeMode.BROWSE, None.instance());
```

Comment:
The only thing formerly receiving mode change notifications was `CanopusFeature`, so we can do entirely away with this.

Reply:
This is true and yet I'm hesitant about getting rid of it.  I guess because it was such a hassle getting it all to work.  But you're right, it's probably best to delete it if we aren't using it.

Reply:
Hopefully we won't need them again but I can imagine the effort to get this right.